main.go

```diff
@@ -81,13 +81,28 @@ var (
 		},
 		[]string{"remote"},
 	)
+	writeErrors = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "storage_bigquery_write_error_count",
+			Help: "Total number of write errors to Bigquery.",
+		},
+	)
+	readErrors = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "storage_bigquery_read_error_count",
```

Comment:
Same here. Consider naming this metric `storage_bigquery_read_error_total`

Reply:
I think errors should be plural. it should be `storage_bigquery_read_errors_total`. I missed this the first time I reviewed it.